pytest-playwright-asyncio/pytest_playwright_asyncio/pytest_playwright.py

```diff
@@ -68,6 +68,14 @@ def _pw_artifacts_folder() -> Generator[tempfile.TemporaryDirectory, None, None]
         pass
 
 
+@pytest.fixture(scope="session", autouse=True)
+def _check_sync_async_incompatibility(pytestconfig: Any) -> None:
```

Comment:
I think I'd instead prefer putting it into the `pytest_addoption` fixture - wdyt? That should work as well and seems very early on when the plugin gets loaded.